specs/experimental/op-stack-manager.md

```diff
@@ -0,0 +1,261 @@
+# OP Stack Manager
+
+[`op-contracts/v1.3.0`]: https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.3.0
+[Optimism Monorepo releases]: https://github.com/ethereum-optimism/optimism/releases
+[contract releases]: https://github.com/ethereum-optimism/optimism/blob/main/packages/contracts-bedrock/VERSIONING.md
+[interop specification]: ../interop/overview.md
+[ethereum-optimism/specs#94]: https://github.com/ethereum-optimism/specs/issues/94
+[admin roles]: https://specs.optimism.io/protocol/configurability.html#admin-roles
+[`SystemConfig` contract]: https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/L1/SystemConfig.sol
+[`L2OutputOracle` contract]: https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/L1/L2OutputOracle.sol
+[standard configuration]: ../protocol/configurability.md
+[superchain registry]: https://github.com/ethereum-optimism/superchain-registry
+[ethereum-lists/chains]: https://github.com/ethereum-lists/chains
+[Batch Inbox]: ../protocol/configurability.md#consensus-parameters
+
+The OP Stack Manager is a contract that deploys the L1 contracts for an OP Stack chain in a single
+transaction. It provides a minimal set of user-configurable parameters to ensure that the resulting
+chain meets the [standard configuration] requirements.
+
+The version deployed is always a governance-approved contract release. The set
+of governance approved [contract releases] can be found on the
+[Optimism Monorepo releases] page, and is the set of releases named
+`op-contracts/vX.Y.Z`.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+**Table of Contents**
+
+- [OP Stack Manager](#op-stack-manager)
+  - [Deployment](#deployment)
+  - [Interface](#interface)
+    - [`deploy`](#deploy)
+    - [Getter Methods](#getter-methods)
+  - [Implementation](#implementation)
+    - [Batch Inbox Address](#batch-inbox-address)
+    - [Contract Deployments](#contract-deployments)
+  - [Security Considerations](#security-considerations)
+    - [Chain ID Source of Truth](#chain-id-source-of-truth)
+    - [Chain ID Frontrunning](#chain-id-frontrunning)
+    - [Chain ID Value](#chain-id-value)
+    - [Proxy Admin Owner](#proxy-admin-owner)
+    - [Upgradeability (ABI Changes)](#upgradeability-abi-changes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Deployment
+
+The OP Stack Manager is deployed at `0xTODO`. It can be deployed as follows:
+
+TODO.
+
+## Interface
+
+Version 1.0.0 of the OP Stack Manager deploys the [`op-contracts/v1.3.0`]
+contracts release.
```

Comment:
This is the MCP release. This will likely change to the `op-contracts/v1.4.0` release, which is fault proofs. However I believe that decision has not been finalized yet, so I've left this as MCP for now

Reply:
Straight to fault proofs has me so hyped!

Reply:
Updated to FP in c2d0835